src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -480,6 +480,11 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('showEditorTabs', "Controls whether opened editors should show in tabs or not."),
 			'default': true
 		},
+		'workbench.editor.dirtyTabBorder': {
+			'type': 'boolean',
+			'description': nls.localize('showEditorDirtyTabBorder', "Controls whether top border is drawn on dirty file tabs or not."),
```

Comment:
"Controls wether modified tabs are highlighted"

Reply:
I think some people would search for `border` keyword to look it up in settings. It already has `highlight` in setting itself; combined with `border` should provide more searchable result.